### PR TITLE
Add line to ignore saved plots with file extension .png

### PR DESCRIPTION
**Pull request by haileymoore**
Add line to the end of .gitignore that will allow git to not pick up saved plots as a change

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,6 @@
 
 # OS files
 .DS_Store
+
+#Ignore saved plot files
+*.png
```
Comment (bpbond) [inline .gitignore]: Best practice is to end the file with a blank line please.

**Review by bpbond — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (bpbond) [reply .gitignore]: Also (I'm picky about these kind of things) please add a space after `#`

Comment (haileymoore) [reply .gitignore]: These have been changed and committed to the branch.

**Review by bpbond — APPROVED** [review]
Nice job @hmoore28

**Pull request merged**